data/src/stream.rs

```diff
@@ -69,10 +69,21 @@ struct Stream {
     receiver: mpsc::Receiver<proto::Message>,
 }
 
-pub async fn run(
+pub fn run(
     server: server::Entry,
     proxy: Option<config::Proxy>,
-    mut sender: mpsc::Sender<Update>,
+) -> impl futures::Stream<Item = Update> {
+    let (sender, receiver) = mpsc::unbounded();
+
+    let runner = stream::once(_run(server, proxy, sender)).map(|_| unreachable!());
```

Comment:
I believe the backpressure may still apply, since the bounded `iced` channel is at the end of the stream no matter what and `select` polls in a round-robin fashion.

I _think_ that to truly get rid of the backpressure, you should use `tokio::task::spawn` for the `_run` future here; instead of composing the streams.